javacutil/src/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -247,6 +247,18 @@ public static VariableElement findFieldInType(TypeElement type, String name) {
         return null;
     }
 
+    /**
+     * Returns the elements of the fields whose simple names are {@code names} and are declared in
```

Comment:
I think it should only return non-private fields, because private fields don't exist in the subclass so it doesn't seem to make sense for the subclass to specify their behavior.  Does that sound right to you?